syntax = "proto3";
package fixbug;
option cc_generic_services = true;

message ResultCode{
    int32 errorcode = 1;
    bytes errormsg = 2;
}
message FriendlistRequest
{
    uint32 userid=1;
}
message  FriendlistResponse
{   ResultCode result=1;
    repeated bytes friends=2;
    


}
service FriendServiceRpc{
    rpc GetFriendlist(FriendlistRequest) returns (FriendlistResponse);
   
}